keywords.txt

```diff
@@ -19,6 +20,8 @@ setBiDirMotorSpeed	KEYWORD2
 configureBLDCMotor	KEYWORD2
 setBLDCmotorRPMspeed	KEYWORD2
 setBLDCDutyCyclespeed	KEYWORD2
+setHallBLDCmotorRPMspeed	KEYWORD2
```

Comment:
As mentioned... There might be some API functions still not added here?

Reply:
One was missing yes, I included it as well.